src/protocol/boolean/random_bits_generator.rs

```diff
@@ -36,13 +68,19 @@ where
 {
     #[must_use]
     #[allow(clippy::needless_pass_by_value)] // TODO: pending resolution of TotalRecords::Indeterminate
```

Comment:
Is this Clippy error suppression still necessary?

Reply:
unfortunately yes, we still call only methods that take `&self`